apps/hash-frontend/src/shared/command-bar.tsx

```diff
@@ -197,14 +203,28 @@ const doesIdentifierMatchKeyboardEvent = (
   error.which === identifier ||
   error.charCode === identifier;
 
-export const CommandBar = () => {
+export const CommandBar: FunctionComponent<{
+  namespaceOwnedById: OwnedById;
+}> = ({ namespaceOwnedById }) => {
   const popupState = usePopupState({
     popupId: "kbar",
     variant: "popover",
   });
 
   const router = useRouter();
 
+  const [createUntitledPage] = useCreatePage(namespaceOwnedById);
+  const { data: accountPages } = useAccountPages(namespaceOwnedById);
+
+  const lastRootPageIndex = useMemo(() => {
+    const rootPages = accountPages
+      .filter(({ parentPageEntityId }) => parentPageEntityId === null)
+      .map(({ index }) => index)
+      .sort();
+
+    return rootPages[rootPages.length - 1] ?? null;
+  }, [accountPages]);
```

Comment:
Can we add `lastRootPageIndex` to the return of `useAccountPages`? 

It could be calculated in the same `map` that's already happening there and saves repeating this in multiple places

Reply:
Sure, have moved it into the hook in https://github.com/hashintel/hash/pull/2581/commits/c6a68eb58ada6d69197a664263517ce1f2bb5b15